modules/virt-connecting-vm-secondarynw-using-fqdn.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc
+
+:_content-type: PROCEDURE
+[id="virt-connecting-vm-secondarynw-fqdn_{context}"]
+= Connecting to a virtual machine on a secondary network by using the cluster FQDN
+
+You can access a virtual machine (VM) that is attached to a secondary network interface from outside the cluster by using the fully qualified domain name (FQDN) of the cluster.
+
+.Prerequisites
+* The QEMU guest agent must be running on the virtual machine.
+* The IP address of the VM that you want to connect to, by using a DNS client, must be public.
+* You have configured the DNS server for secondary networks.
+* You have retrieved the fully qualified domain name (FQDN) of the cluster.
+
+.Procedure
```

Comment:
This step that was part of the google doc is missing - 

Point to the kube secondary DNS
A) Locally, enter the DNS (10.46.41.94) into your resolv.conf
B) Globally, this can be done by editing your enterprise (as in company's IT's DNS server, not the kube DNS) DNS server entries, adding:

```
vm.cnv.engineering.redhat.com. IN NS ns.vm.cnv.engineering.redhat.com.
ns.vm.cnv.engineering.redhat.com. IN A 10.46.41.94
```